Sources/BraintreeThreeDSecure/BTThreeDSecurePostBody.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+
+/// The POST body for `v1/payment_methods/\(urlSafeNonce)/three_d_secure/lookup`
+struct BTThreeDSecurePostBody: Encodable {
```

Comment:
For the POST Encodable model, we are omitting the 'BT' prefix. I think we could omit it here as well.

```suggestion
struct ThreeDSecurePOSTBody: Encodable {
```